019_VowelOrConsonent.js

```diff
@@ -0,0 +1,20 @@
+vowelOrConsonent = (ch) => {
+  if (
+    ch === "a" ||
+    ch === "A" ||
+    ch === "e" ||
+    ch === "E" ||
+    ch === "i" ||
+    ch === "I" ||
+    ch === "o" ||
+    ch === "O" ||
+    ch === "u" ||
+    ch === "U"
+  ) {
+    console.log("It is a vowel");
+  } else {
+    console.log("It is a consonent");
+  }
+  return 1;
```

Comment:
why is it returning 1

Reply:
otherwise, it was showing undefined after the results. I will put return inside the conditional statement now